packages/ui/src/2_molecules/NavMenuItem/NavMenuItem.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { FC, ReactNode, useMemo, useRef } from 'react';
+
+import classNames from 'classnames';
+
+import { Badge } from '../../1_atoms/Badge/Badge';
+import styles from './NavMenuItem.module.css';
+
+const MAX_NOTIFICATIONS_COUNT = 99;
+
+type NavMenuItemProps = {
+  count?: number;
+  onClick?: () => void;
+  children: ReactNode;
+  isActive?: boolean;
+  className?: string;
+  dataActionId?: string;
+};
+
+export const NavMenuItem: FC<NavMenuItemProps> = ({
+  count,
+  onClick,
+  children,
+  isActive,
+  className,
+  dataActionId,
+}) => {
+  const ref = useRef<HTMLButtonElement>(null);
+  const formattedCount = useMemo(
+    () =>
+      count && count > MAX_NOTIFICATIONS_COUNT
+        ? MAX_NOTIFICATIONS_COUNT
+        : count,
+    [count],
+  );
```

Comment:
```suggestion
  const formattedCount = useMemo(
    () =>
      count && count > MAX_NOTIFICATIONS_COUNT
        ? `${MAX_NOTIFICATIONS_COUNT}+`
        : count,
    [count],
  );
```

Reply:
it's better to leave it without `+` because we use fixed `width` and `height` in the Badge component, or I need to adjust the Badge component according to this

Reply:
But it won't work as intended without `+`. It's written in the spec and it also doesn't make logical sense to display let's say `99` if you have `1000` notifications. If there are changes needed in the Badge component, do them pls.